cypress/fixtures/modules/discovery.json

```diff
@@ -432,6 +432,1344 @@
     },
     "HOME_INTENT": {
         "appId": "foo",
-        "intent": { "action": "home", "context": { "source": "voice" } }
-      }
+        "intent": {
+            "action": "home",
+            "context": {
+                "source": "voice"
+            }
+        }
+    },
+    "USERINTEREST_WITH_TYPE_INTEREST_REASON_PLAYLIST_ENTITY_PROGRAM": {
+        "type": "interest",
+        "reason": "playlist",
+        "entity": {
+            "identifiers": {
+                "entityId": "345",
+                "entityType": "program",
+                "programType": "movie"
+            },
+            "info": {
+                "title": "Cool Runnings",
```

Comment:
We need to remove actual movie names and synopsis and keep dummy values here

Reply:
corrected title names, synopsis is dummy one already